tests/dummy/config/environment.js

```diff
@@ -39,7 +39,7 @@ module.exports = function(environment) {
     ENV.APP.rootElement = '#ember-testing';
   }
 
-  if (environment === 'production') {
+  if(environment === 'production'){
```

Comment:
One space required after "if" keyword<br>One (or more) spaces required before opening brace for block expressions